__tests__/package.spec.js

```diff
@@ -11,25 +11,9 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
-
-const path = require('path');
-const util = require('util');
-const exec = util.promisify(require('child_process').exec);
 const packageJson = require('../package.json');
 
 describe('package.json', () => {
-  it('should not have peer dep warnings', async () => {
-    expect.assertions(2);
-    // ex:
-    // npm ERR! peer dep missing: eslint@<2.3.0, required by babel-eslint@5.0.4
-    const { stdout, stderr } = exec('npm ls', {
```

Comment:
I added the `async` keyword in #61 to fix this test, which might be the conflict source. Rebasing against `main` should resolve it.

Reply:
fixed inline on github